swagger_parser/test/parser/schema/enum_name_value_class.json

```diff
@@ -0,0 +1,52 @@
+{
+    "openapi": "3.0.0",
+    "paths": {},
+    "components": {
+      "schemas": {
```

Comment:
There is no test for this scheme, I added an e2e test to generate enum via oneOf, there are some problems with the generation, will need to create the correct expected_files, can you take a look?

Reply:
I'm at a loss as to why it isn't working. I added what it will generate, but then it freaks out. Any help here would be appreciated.